### PR TITLE
Add ONT test dataset for metagenomics workflow

### PR DESCRIPTION
**Pull request by elizabethmcd**
This pull request adds a test dataset for the metagenomics workflow with subsampled ONT reads. The README describes how the subsampled ONT fastq files were generated, because the assembly step with Flye will fail if you don't have any reads that assemble. Using previous assemblies that were performed from these reads, reads were mapped with minimap2 and the reads mapping to a select number of contigs were selected to comprise the subset fastq files. This has been validated with the metagenomics workflow with `--platform nanopore`: 

```
[a7/3cb290] process > ARCADIASCIENCE_METAGENOMICS:NANOPORE:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet_test.csv) [100%] 1 of 1 ✔
[04/24197b] process > ARCADIASCIENCE_METAGENOMICS:NANOPORE:PORECHOP_ABI (oma_T1)                                [100%] 1 of 1 ✔
[e9/f9fb78] process > ARCADIASCIENCE_METAGENOMICS:NANOPORE:FLYE (oma_T1)                                        [100%] 1 of 1 ✔
[85/3c79e2] process > ARCADIASCIENCE_METAGENOMICS:NANOPORE:CUSTOM_DUMPSOFTWAREVERSIONS (1)                      [100%] 1 of 1 ✔
Completed at: 23-Jan-2023 15:35:24
Duration    : 2m 13s
CPU hours   : 0.1
Succeeded   : 4
```

### CODE REVIEW

File: metagenomics/ont/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# cheese-ont-metagenomes
+
+## Information & Purpose
+
+These are two subsampled Nanopore metagenomes sequenced from two different cheeses that were sampled at 8 and 12 weeks, respectively. The metagenomes were sequenced using the Nanopore kit [].
```
Comment (taylorreiter) [inline metagenomics/ont/README.md]: do you need to add which nanopore kit was you used?

File: metagenomics/ont/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+contig_1006	8498	6
+contig_1007	2473	13
+contig_1008	19984	11
+contig_1009	6925	8
+contig_1010	2560	6
+contig_1011	3087	16
+contig_1012	11436	11
+```
+
+### Retrieve Corresponding Reads that makeup candidate contigs
```
Comment (taylorreiter) [inline metagenomics/ont/README.md]:
```suggestion
### Retrieve Corresponding Reads that make up candidate contigs
```

File: metagenomics/ont/samplesheet_test.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+sample,fastq_1,fastq_2
+oma,https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/ont/oma_subset_reads.fq.gz,
+eligo,https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/ont/eligo_subset_reads.fq.gz,
```
Comment (taylorreiter) [inline metagenomics/ont/samplesheet_test.csv]: is it ok to have the cheese names publicly represented in the sample name and fq file name?

**Review by taylorreiter — COMMENTED** [review]
Commenting because I have a few questions. In addition to the inline review comments, how come this PR deletes the hifi2genome test data files?

File: metagenomics/ont/README.md (hunk 1 above)
Comment (elizabethmcd) [reply metagenomics/ont/README.md]: ah yes I need to ask Emily, forgot I had this placeholder here

**Comment by elizabethmcd** [comment]
> Commenting because I have a few questions. In addition to the inline review comments, how come this PR deletes the hifi2genome test data files?

This deletes the hifi2genome because while in the process of doing this validation work I realized the hifi2genome will also fail in the same way in that the test dataset has to be made up of reads that actually assemble together. I probably should have removed that directory as part of the next PR with those modifications but here we are.

File: metagenomics/ont/samplesheet_test.csv (hunk 3 above)
Comment (elizabethmcd) [reply metagenomics/ont/samplesheet_test.csv]: I should ask Emily, easy to change if not

File: metagenomics/ont/README.md (hunk 1 above)
Comment (elizabethmcd) [reply metagenomics/ont/README.md]: updated

File: metagenomics/ont/samplesheet_test.csv (hunk 3 above)
Comment (elizabethmcd) [reply metagenomics/ont/samplesheet_test.csv]: updated

**Review by taylorreiter — APPROVED** [review]
LGTM!

**Pull request merged**